src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java

```diff
@@ -46,12 +46,16 @@ public FtpClient(
     }
     // endregion
 
-    public void upload(PdfDoc pdfDoc) {
+    public void upload(PdfDoc pdfDoc, boolean isSmokeTest) {
```

Comment:
How about `isSmokeTestData`/`isSmokeTestDoc` to make it obvious we're not running in any spacial mode, just deal with special data? Not strong about it, up to you.